deploy/helm/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+## [2.2.0-alpha.3] - 2023-03-07
```

Comment:
Let's keep this as "Unreleased" for now. There are other changes, that need to be merged into this PR anyway.

Reply:
Removed